src/text-link.css.ts

```diff
@@ -73,7 +73,3 @@ export const variants = styleVariants({
         },
     ],
 });
-
-export const small = style({
-    fontSize: 14,
```

Comment:
moved to inline styles because the `BaseTouchable`  overrides this to `'inherit'`